templates/vue/src/components/modals/common/FileUpload.vue

```diff
@@ -110,14 +110,27 @@
     <b-alert v-if="error" show variant="danger">
       File upload failed: {{ error.message || error.data.message }}
     </b-alert>
-    <b-alert v-else-if="!confirmedUpload" show variant="success">
+    <b-alert v-else-if="!confirmedUploadToServer" show variant="success">
       <b-row>
         <b-col cols="auto" class="upload-label mr-auto text-muted">
           Upload completed successfully. Press the "Submit" button to save your
           changes.
         </b-col>
         <b-col cols="auto">
-          <b-button size="sm" variant="secondary" @click="confirmUpload">
+          <b-button size="sm" variant="secondary" @click="confirmUploadToServer">
+            OK
+          </b-button>
+        </b-col>
+      </b-row>
+    </b-alert>
+    <b-alert v-else-if="!confirmedUploadToKaltura" show variant="success">
+      <b-row>
+        <b-col cols="auto" class="upload-label mr-auto text-muted">
+          File will now upload to Kaltura in the background! You can now publish the
+          node and it will become available when the file is finish uploading.
```

Comment:
I don't think we need to include the user in this level of detail. We can just let them know the video they uploaded is being processed in the background. No need to mention Kaltura.